Services/WalletConnectorService/Sources/WalletConnectorService.swift

```diff
@@ -15,9 +24,9 @@ public final class WalletConnectorService: Sendable {
     }
 }
 
-// MARK: - Public
+// MARK: - WalletConnectorService
 
-extension WalletConnectorService {
+extension WalletConnectorServiceImpl: WalletConnectorService {
```

Comment:
```suggestion
extension WalletConnectorService: WalletConnectorServiceable {
```

this is Swift, not Java